src/webgpu/web_platform/copyToTexture/canvas.spec.ts

```diff
@@ -650,6 +650,75 @@ g.test('copy_contents_from_gpu_context_canvas')
     });
   });
 
+g.test('copy_contents_from_ImageBitmapRendering_context_canvas')
```

Comment:
nit: the context type string is 'bitmaprenderer'
```suggestion
g.test('copy_contents_from_bitmaprenderer_context_canvas')
```